### PR TITLE
[Response Datastore] Additional Debugger added for investigate issue 2339

### PR DESCRIPTION
**Pull request by aswinijena100**
Additional debugger added to investigate the issue #2339

### CODE REVIEW

**Comment by moschetti** [comment]
if this is a single issue debugger, do we need it merged into dev branch?

**Comment by aswinijena100** [comment]
Yes,this PR is only for single issue debugger. We can merge into develop as well

**Review by moschetti — APPROVED** [review]

**Review by moschetti — APPROVED** [review]
Should not need this since issue was found and resolved. But approving in case you need this to do further troubleshooting.

File: response-datastore/response-server-service/src/main/resources/application.properties [hunk 1]
```diff
@@ -13,8 +13,8 @@ spring.main.allow-bean-definition-overriding=true
 #logging.file=${logging.path}/response_datastore_service.log
 #logging.file.max-size=1024KB
 
-logging.level.root=WARN
-logging.level.com.google.cloud.healthcare.fdamystudies=WARN
+logging.level.root=DEBUG
```
Comment (dhanyak-btc) [inline response-datastore/response-server-service/src/main/resources/application.properties:16]: Created Issue #2851 to revert the log level to WARN after troubleshooting the issue.

**Pull request merged**